native/src/components/PoiDetails.tsx

```diff
@@ -44,14 +43,14 @@ const StyledCategory = styled.Text`
 
 type PoiDetailsProps = {
   poi: PoiModel
-  poiFeature: GeoJsonPoi
   language: string
+  distance: number | null
 }
 
-const PoiDetails = ({ poi, poiFeature, language }: PoiDetailsProps): ReactElement => {
+const PoiDetails = ({ poi, language, distance }: PoiDetailsProps): ReactElement => {
   const { t } = useTranslation('pois')
-  const thumbnail = poiFeature.thumbnail?.replace('-150x150', '') ?? PoiThumbnailPlaceholderLarge
-  const distance = poiFeature.distance
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  const thumbnail = poi.thumbnail?.replace('-150x150', '') ?? PoiThumbnailPlaceholderLarge
```

Comment:
Sadly our types here are wrong, `thumbnail` may very well be null (see https://digitalfabrik.github.io/integreat-cms/api-docs.html#locations).

Reply:
Would it make sense to open a ticket to fix them?

Reply:
Ahh sorry, included it in this issue: #2635.